agenta-backend/agenta_backend/models/api/api_models.py

```diff
@@ -44,7 +44,6 @@ class SaveConfigPayload(BaseModel):
     base_id: str
     config_name: str
     parameters: Dict[str, Any]
-    overwrite: bool
```

Comment:
Why are we removing the overwrite option? I think it's out of scope for this PR, no? The decision on how to re-design this interface needs to be done somewhere else imho